src/main/java/bio/terra/common/iam/SamUserFactory.java

```diff
@@ -28,16 +29,16 @@
 @Component
 public class SamUserFactory {
   final BearerTokenFactory bearerTokenFactory;
-  final OkHttpClient httpClient =
-      new ApiClient()
-          .getHttpClient()
-          .newBuilder()
-          .addInterceptor(new OkHttpClientTracingInterceptor(Tracing.getTracer()))
-          .build();
+  final OkHttpClient httpClient;
 
-  @Autowired
-  public SamUserFactory(BearerTokenFactory bearerTokenFactory) {
+  @Autowired(required = false)
+  public SamUserFactory(
+      BearerTokenFactory bearerTokenFactory, Optional<OpenTelemetry> openTelemetry) {
```

Comment:
Similar to declaring a constructor arg `@Nullable`, declaring it as `Optional` should also set `required = false` so you don't need to change the annotation. If you leave `required = false`, that makes `BearerTokenFactory` nullable too which I don't think you want to do.
```suggestion
  @Autowired
  public SamUserFactory(
      BearerTokenFactory bearerTokenFactory, Optional<OpenTelemetry> openTelemetry) {
```